backend/app/testbot/llm.py

```
@@ -0,0 +1,32 @@
+        try:
+            result = response.json()
+            if "candidates" in result and len(result["candidates"]) > 0:
+                candidate = result["candidates"][0]
+                if "content" in candidate and "parts" in candidate["content"]:
+                    answer_text = candidate["content"]["parts"][0].get("text", "").strip()
+                    return answer_text if answer_text else "No output returned."
+                else:
+                    return "No content found in candidate."
+            else:
+                return json.dumps(result, indent=2)
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)